File: PRAK405-2310817220016-NailaHanifah.c
#include <stdio.h>

int main() {
    int n, kelipatan;

    printf("Input: ");
    scanf("%d %d", &n, &kelipatan);

    int jumlah = 0;

    for (int i = 1; i <= n; i++) {
        int hasilBaris = 0;

        for (int j = i; j >= 1; j--) {
            int nilai = j * kelipatan;
            printf("(%d * %d)", j, kelipatan);
            hasilBaris += nilai;

            if (j > 1) {
                printf(" + ");
            }
        }
        printf(" = %d\n", hasilBaris);
        jumlah += hasilBaris;
    }
    printf("%d\n", jumlah);

    return 0;
}

File: PRAK405-2310817220016-NailaHanifah.py
def hitung_kelipatan(baris, kelipatan):
    total_per_baris = []

    for i in range(1, baris + 1):
        row_total = 0
        row_calculation = []

        for j in range(i, 0, -1):
            nilai = j * kelipatan
            row_calculation.append(f"({j} * {kelipatan})")
            row_total += nilai

        print(f"{' + '.join(row_calculation)} = {row_total}")
        total_per_baris.append(row_total)

    return total_per_baris

def main():
    baris, kelipatan = map(int, input("Input: ").split())
    total_per_baris = hitung_kelipatan(baris, kelipatan)
    
    print(sum(total_per_baris))

if __name__ == "__main__":
    main()